workitem_test.go

```diff
@@ -19,14 +18,12 @@ import (
 var db *gorm.DB
 
 func TestMain(m *testing.M) {
-	var dbhost *string = flag.String("dbhost", "", "-dbhost <hostname>")
-	flag.Parse()
-	if "" == *dbhost {
-		flag.Usage()
-		os.Exit(-1)
+	dbhost := os.Getenv("ALMIGHTY_DB_HOST")
+	if "" == dbhost {
+		panic("The environment variable ALMIGHTY_DB_HOST is not specified or empty.")
```

Comment:
I thought about this as well but we only have [`testing.M`](https://golang.org/pkg/testing/#M) here and that has no such method.